Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN yum upgrade -y
 RUN amazon-linux-extras install -y epel && yum install -y libASL --skip-broken
 RUN yum install -y  \
       glibc-devel \
+      libyaml-devel \
```

Comment:
we need to add `libyaml` as a runtime dependency as well AFAIK. I tried testing this before- it came up during my record_accessor bug bash remember?